src/drivers/pca9685_pwm_out/PCA9685.cpp

```diff
@@ -79,14 +74,14 @@ int PCA9685::setFreq(float freq)
 	uint16_t realResolution = floorl((float)PCA9685_CLOCK_FREQ / freq);
 
 	if (realResolution < PCA9685_PWM_RES) { // unable to provide enough resolution
-		PX4_DEBUG("frequency too high");
+		PX4_ERR("frequency too high");
```

Comment:
Nitpick, this message probably isn't too useful without more context.

Reply:
This is the whole output:
```
pxh> param set PWM_RATE 10
+ PWM_RATE: curr: 400 -> new: 10
pxh> ERROR [pca9685_pwm_out] frequency too low
ERROR [pca9685_pwm_out] failed to set pwm frequency, fall back to 50Hz
pxh> 

```

Reply:
It's not a big deal, but I think a single message with context would be more useful, especially at bootup if you didn't just try to change `PWM_RATE`.

``` Console
ERROR [pca9685_pwm_out] failed to set PWM frequency to 10 Hz, fall back to 50Hz
```

Reply:
Done.